src/pages/Products/Products.jsx

```diff
@@ -1,3 +1,27 @@
+import { Loading, ProductCard } from "../../components/index";
```

Comment:
you can directly import the components from `../../components`, anything that's exported from an index.js file of a folder, become default exports for that folder

Reply:
Okay Thanks for pointing it out